kpops/components/base_components/pipeline_component.py

```diff
@@ -75,6 +75,18 @@ def __init__(self, **kwargs) -> None:
         self.set_input_topics()
         self.set_output_topics()
 
+    def get_input_topics(self) -> list[str] | None:
```

Comment:
please keep the method order consistent

1. `get_input_topics`
2. `get_extra_input..`
3. `get_output..`
4. `get_extra_output..`